api/scripts/update-users-with-shouldValidateTermsOfService-to-true.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('lodash');
+const Bookshelf = require('../lib/infrastructure/bookshelf');
+
+const BookshelfUser = require('../lib/infrastructure/data/user');
+
+async function updateShouldValidateTermsOfServiceWhenUsersHaveNotSamlIdAndNotAssociatedWithSchoolingRegistration() {
```

Comment:
Le nom est complet mais ça fait sacrément long.
Je n'ai pas d'autres idées pour le dommage mais s'il y a moyen de trouver un nom plus court ce serait cool.

Reply:
On peut déjà remplacer `should` par `must` et en plus ça serait plus correct 😉

Reply:
@octo-thlo s'en occupe

Reply:
Le "must" peut aussi être adopté poru le nom de la colonne.
Penser à modifier le libellé des commit en conséquence

Reply:
En le relisant je me rends compte que ce nom est inapproprié parce qu'il détaille le _how_ de la fonction et pas le _what_.

Par exemple :

```suggestion
async function markUsersRequiringTermsOfServiceValidationForRevalidation() {
```

Reply:
Colonne / variables / commit renommés en "must"

Reply:
Chouette, on arrive à la même conclusion sur le nom du script. Tel quel, le nom ne fournit pas d'abstraction, il pourrait s'appeler 1, ça serait moins casse-pied, et si l'implémentation change, au moins son nom ne nous induira pas en erreur. 
J'ai aussi renommé le script en mark-users-for-TermsOfService-revalidation

Reply:
Corrigé